docs/docker/README.md

```diff
@@ -162,10 +161,27 @@ virtual machine in which all Docker processes are running.
 cBioPortal can now be reached at <http://localhost:8081/>
 
 Activity of Docker containers can be seen with:
+
 ```
 docker ps -a
 ```
 
+## A note on versioning ##
+
+For production you might want to deploy a specific docker image tag, instead of
+the `latest` image. The version can be seen in the footer of the home page. The
+various image versions can be found here:
+https://hub.docker.com/r/cbioportal/cbioportal/tags. You can also get the
+latest version programmatically like this:
+
+```
+LATEST_VERSION=$(curl --silent "https://api.github.com/repos/cBioPortal/cbioportal/releases/latest" \
+    | grep "tag_name" \
+    | cut -d'"' -f4 \
+    | cut -dv -f2)
+echo $LATEST_VERSION
```

Comment:
I would just remove this part...but that's just because I see no reason why one would like to do this programmatically (except for beta portals that are automatically updated...but these could also just run on :latest)

Reply:
not really - just figured that if you deployed `latest`, it's not super straightforward to figure out what tag that corresponds to. Cause if you go to the docker hub tag page you see a zillion images. If you go to the github page with the latest release, you have to remove the `v` prefix to make it work. This way you can copy+paste a command, so seemed easier